frontend/lang/en.js

```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Title',
```

Comment:
Idem

Reply:
what does idem mean?

Reply:
It means "as above", in this case the label title is still used and can not be removed